server/routes/yourCompletedReports/yourCompletedReports.ts

```diff
@@ -41,7 +41,9 @@ export default class YourCompletedReportsRoutes {
     const filter = filterFromUiFilter(uiFilter)
     const results = await this.reportedAdjudicationsService.getYourCompletedAdjudications(
       res.locals.user,
-      filter,
+      /* eslint-disable */
+      filter as any, // TODOO - NN-4461
+      /* eslint-enable */
```

Comment:
As with the other comment, we shouldn't disable eslint for this - the type needs changing somewhere - I think it will be sorted if you change the type I suggested above